src/App.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import './App.css';
+import BooksList from './components/BooksList';
```

Comment:
- [ ] `App.js` should be placed in src/components directory since it's a presentational component.